test/worker_threads.test.js

```diff
@@ -0,0 +1,18 @@
+import Test from './common.js';
+import { Worker } from '../nodelibs/browser/worker_threads.js';
+import { once } from '../nodelibs/browser/events.js';
+
+const test = new Test();
+
+const worker = new Worker(`import { parentPort } from '../nodelibs/browser/worker_threads.js';
+import { once } from '../nodelibs/browser/events.js';
+
+parentPort.postMessage('hello');
+parentPort.postMessage(\`received: \${(await once(parentPort, 'message'))[0]}\`);
+`, { eval: true });
+worker.postMessage('hello2');
+
+test.assertObjectEqual(async () => await once(worker, 'message'), ['hello']);
+test.assertObjectEqual(async () => await once(worker, 'message'), ['received: hello2']);
```

Comment:
Then this should just work:

```suggestion
{
  const msg = await once(worker, 'message');
  test.assertObjectEqual(msg, ['hello']);
}
{
  const msg = await once(worker, 'message');
  test.assertObjectEqual(msg, ['recieved: hello2']);
}
```

Reply:
`() => msg` instead.
We lose a little bit of the verbosity. Maybe we can just remove functions in asserts first argument?

Reply:
Personally I find the less invisible logic in tests the better.

Reply:
What do you mean by that?

Reply:
I agree it's simpler not to have assert functions.